TCC/Program.cs

```diff
@@ -1,14 +1,18 @@
 ﻿using System;
 using System.Collections.Concurrent;
 using System.Threading;
+using System.Threading.Tasks;
 using TCC.Lib;
+using TCC.Lib.Dependencies;
+using TCC.Lib.Options;
+using TCC.Lib.Parser;
+using TCC.Lib.Process;
 
 namespace TCC
 {
     class Program
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a 'protected' constructor or the 'static' keyword to the class declaration. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1118)